### PR TITLE
[CMS] ButterCMS Integration Guide

### PR DESCRIPTION
**Pull request by Jutanium**
- New or updated content

### Description
- Adds an integration guide for Butter - it doesn't teach you how to _use_ Butter's interface, but how to pull content into your Astro project.
- Renames `buttercms.md` and `payload.md` to reflect what they call themselves

### CODE REVIEW

File: src/pages/en/guides/cms/buttercms.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+setup: |
+    import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+stub: false
+---
+
+ButterCMS is a headless CMS and blog engine that allows you to publish structured content to use in your project.
+
+## Integrating with Astro
+
+In this section, we'll use the ButterCMS SDK to bring your ButterCMS data into your Astro project.
```
Comment (sarah11918) [inline src/pages/en/guides/cms/buttercms.md:14]:
```suggestion
In this section, we'll use the [ButterCMS SDK](https://www.npmjs.com/package/buttercms) to bring your data into your Astro project.
```
ButterCMS twice sounded a little clunky? Also included a link to the SDK. maybe?

File: src/pages/en/guides/cms/buttercms.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+### Setup
+
+1. Create a `.env` file in the root of your project and add your API token as an environment variable:
+
+```ini title=".env"
+BUTTER_TOKEN=YOUR_API_TOKEN_HERE
+```
+
+:::tip
+Check out [Using environment variables](/en/guides/environment-variables/) for more information about environment variables and `.env` files in Astro.
```
Comment (sarah11918) [inline src/pages/en/guides/cms/buttercms.md:35]:
```suggestion
See more about [using environment variables](/en/guides/environment-variables/) and `.env` files in Astro.
```
This is a little awkward, but I'd try to work it into the sentence, not state environment variables twice.

File: src/pages/en/guides/cms/buttercms.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+```js title="src/lib/buttercms.js"
+import Butter from "buttercms";
+
+export const butterClient = Butter(import.meta.env.BUTTER_TOKEN);
+```
+
+This authenticates the SDK using your API Token and exports it to be used across your project.
+
+### Fetching Data
+To fetch content, import this client and use one of its `retrieve` functions. In this example, we [retrieve a collection](https://buttercms.com/docs/api/#retrieve-a-collection) that has three fields: a short text `name`, a number `price`, and a WYSIWYG `description`.
```
Comment (sarah11918) [inline src/pages/en/guides/cms/buttercms.md:68]:
```suggestion
To fetch content, import this client and use one of its `retrieve` functions. 

In this example, we [retrieve a collection](https://buttercms.com/docs/api/#retrieve-a-collection) that has three fields: a short text `name`, a number `price`, and a WYSIWYG `description`. 
```
See how this looks. Wherever it's not too clunky, I like to start the "for example" on its own line, separate from a more general instruction.

File: src/pages/en/guides/cms/buttercms.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+	{items.map(item => <div>
+		<h2>{item.name} - ${item.price}</h2>
+		<p set:html={item.description}></p>
+	</div>)}
+</body>
+```
+
+The interface mirrors the field types. The WYSIWYG `description` field loads as a string of HTML, and [`set:html`](/en/reference/directives-reference/#sethtml) lets you render it.
+
+Similarly, you can [retrieve a page](https://buttercms.com/docs/api/#get-a-single-page):
```
Comment (sarah11918) [inline src/pages/en/guides/cms/buttercms.md:93]:
```suggestion
Similarly, you can [retrieve a page](https://buttercms.com/docs/api/#get-a-single-page) and return data to display a blog post:
```
That sentence felt a little flat without an "ending." I don't think this needs to be a technical description like the example above. Rather, if you can just say anything generic, e.g. purpose, I think it reads better.

**Review by sarah11918 — COMMENTED** [review]
Hey @Jutanium - So excited to see these CMS guides filling out!  🎉 

Just left a few comments for your consideration... but this is looking pretty good to go!

File: src/pages/en/guides/cms/buttercms.md (hunk 2 above)
Comment (Jutanium) [reply src/pages/en/guides/cms/buttercms.md:35]: I'll update this language in the Contentful guide too!

**Pull request merged**